### DISCUSSION TITLE
No audio coming in Ableton !

### DISCUSSION THREAD

**Discussion by personanonymous** [ExistentialAudio/BlackHole#724] [Q&A]
So I downloaded the programme step by step following this tutorial: https://www.youtube.com/watch?v=F67Jj5QCct8&ab_channel=SeedtoStage

However, when I put blackhole as my input for ableton and play a youtube video, A) i dont hear the youtube video which is kinda fine, but more importantly B) I have no audio coming into ableton into audio tracks.

What gives?

**Comment by devinroth** [comment]
Yeah that doesn’t make sense. Can you do a test with Audacity?

Devin

**Comment by dylz04** [comment]
did you fix this problem ? i am having the same no audio through speakers and no input in ableton. Have tested with audacity still no luck. Tried installing from homebrew and with official installer. Macos 1015.7 any ideas ? thanks

**Comment by devinroth** [comment]
Are you able to record with other audio devices?

And what audio are you sending to BlackHole.

Devin

**Reply by dylz04** [reply]
i am able to record audio with microphone in audacity. I was just testing with youtube and spotify.

**Comment by devinroth** [comment]
Did you check mute and volume on BlackHole?

Devin

**Comment by dylz04** [comment]
input and output both on full and mute is disabled

**Comment by dylz04** [comment]
just worked on audacity i dont think i had set input to blackhole

**Comment by dylz04** [comment]
is there any way of having sound from my laptop whilst blackhole is selected without a mutli output device ?

**Comment by dylz04** [comment]
nothing coming into ableton now : !

**Comment by dylz04** [comment]
hey it is now working was a problem with apple permissions i had to disable and re enable mic access. 
Thannks
